server/sonar-server/src/main/java/org/sonar/server/es/DefaultIndexSettingsElement.java

```diff
@@ -25,17 +25,16 @@
 import java.util.SortedMap;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.settings.Settings.Builder;
+import org.sonar.server.component.index.ComponentIndexSearchFeature;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.server.component.index.ComponentIndexSearchFeature'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)